src/main/java/seedu/canoe/logic/commands/TrainingCommand.java

```diff
@@ -21,6 +21,7 @@
     public static final String MESSAGE_SUCCESS_TRAINING = "New Training Session created at: %1$s";
     public static final String MESSAGE_DUPLICATE_TRAINING = "There "
             + "already exists a Training Session at this Date and Time";
+    public static final String MESSAGE_PAST_TRAINING = "Trainings can only be scheduled for the future!";
```

Comment:
Maybe "Trainings cannot be scheduled for dates that are past"? Or something like that?